test/Widget/RegionServiceTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Widget;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\Translator;
+
+final class RegionServiceTest extends TestCase
+{
+    private RegionService $regionService;
+
+    public function setUp(): void
+    {
+        $this->regionService = new RegionService(
+            './test/Widget/autocompletemini.json',
```

Comment:
Now that I think again about this name, it rather is just a regions sample file, maybe that is more meaningful as a name?